src/workflows/airqo_etl_utils/bigquery_api.py

```diff
@@ -58,14 +58,14 @@ def get_devices_hourly_data(
         day: datetime,
     ) -> pd.DataFrame:
         query = (
-            f" SELECT {self.hourly_measurements_table}.pm2_5_calibrated_value , "
-            f" {self.hourly_measurements_table}.pm2_5_raw_value ,"
-            f" {self.hourly_measurements_table}.site_id ,"
-            f" {self.hourly_measurements_table}.device_id AS device ,"
-            f" FORMAT_DATETIME('%Y-%m-%d %H:%M:%S', {self.hourly_measurements_table}.timestamp) AS timestamp ,"
-            f" FROM {self.hourly_measurements_table} "
-            f" WHERE DATE({self.hourly_measurements_table}.timestamp) >= '{day.strftime('%Y-%m-%d')}' "
-            f" AND {self.hourly_measurements_table}.pm2_5_raw_value is not null "
+            f" SELECT `{self.hourly_measurements_table}`.pm2_5_calibrated_value , "
+            f" `{self.hourly_measurements_table}`.pm2_5_raw_value ,"
+            f" `{self.hourly_measurements_table}`.site_id ,"
+            f" `{self.hourly_measurements_table}`.device_id AS device ,"
+            f" FORMAT_DATETIME('%Y-%m-%d %H:%M:%S', `{self.hourly_measurements_table}`.timestamp) AS timestamp ,"
+            f" FROM `{self.hourly_measurements_table}` "
```

Comment:
There is unnecessary comma before the `FROM` clause

Correct that to;
`f"FORMAT_DATETIME('%Y-%m-%d %H:%M:%S', `{self.hourly_measurements_table}`.timestamp) AS timestamp "
        f"FROM `{self.hourly_measurements_table}` "`

Reply:
Oh yeah, thanks